v2/graph_type.go

```diff
@@ -10,3 +10,19 @@
 package v2
 // GraphType : The graph type.
 type GraphType string
+
+// List of GraphType
+const (
```

Comment:
This shouldn't be there. Please run the sed commands in the CONTRIBUTING section and re-commit this file